doc/book/admin/access_control.rst

```diff
@@ -1,270 +1,854 @@
 
-The current user can be changed:
+There are two built-in users in Tarantool:
+
+*   'admin' is a user with all available administrative permissions.
+    If the connection uses an :ref:`admin-console port <admin-security>`, the current user is 'admin'.
+    For example, 'admin' is used when connecting to the instance using :ref:`tt connect <tt-connect>` locally using the instance name:
+
+    ..  code-block:: console
+
+        $ tt connect app:instance001
+
+    ..  NOTE::
+
+        To allow remote :ref:`binary port <admin-security>` connections using the 'admin' user, you need to :ref:`set a password <access_control_user_changing_passwords>`.
+
+*   'guest' is a user with minimum permissions used by default for remote :ref:`binary port <admin-security>` connections.
+    For example, 'guest' is used when connecting to the instance using :ref:`tt connect <tt-connect>` using the IP address and port without specifying the name of a user:
+
+    ..  code-block:: console
+
+        $ tt connect 192.168.10.10:3301
+
+
+..  NOTE::
+
+    Information about users is stored in the :ref:`_user <box_space-user>` space.
 
-* For a binary port connection -- with the
-  :ref:`AUTH protocol command <box_protocol-iproto_protocol>`, supported
-  by most clients;
 
-* For an admin-console connection and in a Lua initialization script --
-  with :doc:`/reference/reference_lua/box_session/su`;
 
-* For a binary-port connection invoking a stored function with the CALL command --
-  if the :doc:`SETUID </reference/reference_lua/box_schema/func_create>`
-  property is enabled for the function,
-  Tarantool temporarily replaces the current user with the
-  function’s creator, with all the creator's privileges, during function execution.
 
 .. _authentication-passwords:
 
---------------------------------------------------------------------------------
 Passwords
---------------------------------------------------------------------------------
+~~~~~~~~~
 
-Each user (except 'guest') may have a **password**.
+Any user (except 'guest') may have a password.
 The password is any alphanumeric string.
+Tarantool :ref:`password hashes <enterprise-authentication-protocol>` are stored in the :ref:`_user <box_space-user>` system space.
+So, if the password is '123456', the stored hash is a string like 'a7SDfrdDKRBe5FaN2n3GftLKKtk='.
 
-Tarantool passwords are stored in the :ref:`_user <box_space-user>`
-system space with a
-`cryptographic hash function <https://en.wikipedia.org/wiki/Cryptographic_hash_function>`_
-so that, if the password is ‘x’, the stored hash-password is a long string
-like ‘lL3OvhkIPOKh+Vn9Avlkx69M/Ck=‘.
-Tarantool supports two protocols for authenticating users:
+Tarantool Enterprise Edition allows you to improve database security by enforcing the use of strong passwords, setting up a maximum password age, and so on.
+Learn more from the :ref:`configuration_authentication` topic.
```

Comment:
Thanks, added:

<img width="846" alt="image" src="https://github.com/tarantool/doc/assets/38073144/c73e3390-dea7-40e6-912d-d3e83f1a81b5">

Reply:
> a password is salted with a user-unique salt
Maybe:
a password is salted with randomly generated user-unique data


Actually, I don't think we need to get into details of implementation, password salt is always random and unique, so maybe just say we support it, it may be a required criteria for some organization.

Reply:
Okay, let's remove this sentence - the reader can find this info in a linked page.

Reply:
Ok